resident/resident-service/src/main/java/io/mosip/resident/util/EventEnum.java

```diff
@@ -642,6 +642,13 @@ public enum EventEnum {
 	SEND_OTP_SUCCESS("RES-SER-259", RegistrationConstants.SYSTEM, "send otp success",
 			"send otp is success", "RES-SER", "Residence service", "NO_ID", "NO_ID_TYPE",
 			RegistrationConstants.APPLICATIONID, RegistrationConstants.APPLICATIONNAME),
+	
+	GET_SERVICE_HISTORY_PDF_URL("RES-SER-260", RegistrationConstants.SYSTEM, "get service history pdf",
```

Comment:
Don't refer to the API as PDF as we want to support CSV also in future, just refer to it in all places as 'download service history'